backend/stump_backend/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.6.10-alpine
+
+WORKDIR /usr/src/backend/stump_backend
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# psycopg2 dependencies
+RUN apk update \
+  && apk add postgresql-dev gcc python3-dev musl-dev
+
+# install python dependencies
+COPY ./Pipfile* ./
+RUN pip install --upgrade pip \
+  && pip install pipenv \
+  && pipenv install --system --dev --deploy
```

Comment:
@galbwe Are the dev dependencies needed for the backend build?

Reply:
@coloradocarlos at the moment, there are no development dependencies specified in the Pipfile, so it is not strictly necessary at the moment. I intended the build in ```Dockerfile``` to be used by developers on their machines, so I would expect the --dev flag to be used to install a linter, a unit test library, or any other dependencies from the ```[dev-packages]``` section of ```Pipfile``` that are not strictly needed for production. I think I am going to update the name of ```Dockerfile``` to ```Dockerfile.local``` and add some comments to emphasize this point.